packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -265,6 +266,12 @@ const Dropdown = ({
     };
 
     const handleVisibility = () => {
+        if (typeof onClick === 'function') {
+            onClick();
+
+            return;
```

Comment:
why do we check for onClick as a type of function if its already defined as a function in proptypes? and why do we return after the if check? it wouldn't check the other if statements in the function right?

Reply:
This is to check if we are passing `onClick` since it is not required. Yes, we don't need to run other statements if `onClick` is passed.